### DISCUSSION TITLE
CT24.cz doesnt work

### DISCUSSION THREAD

**Discussion by palunador** [aclap-dev/video-downloadhelper#788] [Help and Q&A] [answered]
### Your question

Download is not working

### Addon & CoApp version and details

My PC: Kubuntu 24.04

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://ct24.ceskatelevize.cz/clanek/domaci/zive-brifink-premiera-fialy-po-jednani-s-generalnim-tajemnikem-nato-349756",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 0x672abc0] Error opening input: Server returned 404 Not Found\nError opening input file https://ivys-vst24-5.o2tv.cz/atip/9f40964b7def0d790efa45a19988c685/1717083461324/subscru/685fedf6-5e2a-40f5-ba6f-2044f1eb2e88/61924494878229533-1080p_5-0-858.mpd.\nError opening input files: Server returned 404 Not Found\n",
    "id": "coapp_failure_ce42ff35-4ae9-46e6-a006-c3ecfb514142",
    "downloadable_id": "downloadable_5415280764396615"
  }
}

**Comment by Mattmjs** [comment, marked as answer]
I tested your page and it works, no error message. It is extremely slow the download meaning it's heavily throttled.
That's the only issue I see.

**Reply by palunador** [reply]
Yes, now is working correctly. I dont know where was the problem. Thank you.
